apidoc/Titanium/Media/Media.yml

```diff
@@ -1749,6 +1767,13 @@ properties:
     default: true
     since: "3.4.0"
     platforms: [ipad]
+  - name: focusMode
+    summary: Constant to indicate focus mode of camera.
+    constants: Titanium.Media.FOCUS_MODE_*
+    type: Number
+    default: Depends on camera
+    platforms: [android, iphone, ipad]
```

Comment:
You don't need to specify the platforms if all three are supported. It will use all by default. Same for the other docs as well.